### ISSUE TITLE
Should warn about mistyped/unknown options in .nengobones.yml

### ISSUE THREAD

**Issue by jgosmann** [nengo/nengo-bones#26]
Otherwise it is easy for settings to not take effect without noticing.

**Comment by drasmuss** [comment]
Yep, this is [on our TODO list](https://github.com/nengo/nengo-bones/blob/master/nengo_bones/config.py#L75)!  I will update here when progress is made.
